### PR TITLE
Docker updates

### LINKED ISSUE #123: When to run collectstatic command
I see there are a mix of auto-generated and manually-created files in `data_registry/static`.

1. Which files are auto-generated?
2. When does an admin need to run collectstatic?
3. Why can't collectstatic be run on the server instead, during deployment?

Found while running:

```bash
find . -type f -not -path '*/migrations/*' -not -path '*/.git/*' -not -path '*/static/img/flags/*' -not -name '*.pyc'
```

### PR DESCRIPTION
**Pull request by yolile**
closes #123 
ref https://github.com/open-contracting/deploy/issues/320

### CODE REVIEW

File: Dockerfile_django [hunk 1]
```diff
@@ -1,14 +1,21 @@
 FROM python:3.8
 
-WORKDIR /usr/src/app
+RUN apt-get update && apt-get install -y gettext
```
Comment (jpmckinney) [inline Dockerfile_django:3]:
```suggestion
RUN apt-get update && apt-get install -y --no-install-recommends \
      gettext \
   && rm -rf /var/lib/apt/lists/*
```

File: Dockerfile_static [hunk 2]
```diff
@@ -1,23 +1,25 @@
-FROM python:3 as build-stage
+# vue statics
+FROM node:16 as build-stage-vue
 
-WORKDIR /usr/src/app
+# For the static files to be generated in workdir/static
+WORKDIR /workdir/data_registry/vue
+COPY data_registry/vue/package*.json ./
+RUN npm ci
```
Comment (jpmckinney) [inline Dockerfile_static:7]: Similar to the change in pelican-frontend

```suggestion
RUN npm install
```

File: .gitignore [hunk 3]
```diff
@@ -2,5 +2,10 @@ db.sqlite3
 
 __pycache__/
 *.pyc
+*.mo
 
 node_modules
+
+static/*
+!static/fonts
+!static/img
```
Comment (jpmckinney) [inline .gitignore:11]: I don't think we want to move the static files to `STATIC_ROOT` (`static`). I think we can leave them where they are, and `collectstatic` will move them here each time.

Comment (yolile) [reply .gitignore]: Done in https://github.com/open-contracting/data-registry/pull/170/commits/c790ee71d5bb4171497a1cac6791327828d71e73

**Pull request merged**